crates/nao/src/lib.rs

```diff
@@ -11,14 +11,40 @@ use color_eyre::{
 use tokio::process::Command;
 
 pub struct Nao {
-    host: Ipv4Addr,
+    pub host: Ipv4Addr,
 }
 
 impl Nao {
     pub fn new(host: Ipv4Addr) -> Self {
         Self { host }
     }
 
+    pub async fn has_stable_os_version(&self) -> bool {
+        let os_stable = "5.2.0";
```

Comment:
@knoellle What do you think about this introduction of a new magic value? Now we need to maintain the version number twice: 1. in the `repository` crate and 2. here in the `nao` crate. We should have a single point of truth. Shared crate? Shared configuration file (reading the configuration file can also be deduplicated in a shared crate)?